docs/src/manual/variables.md

```diff
@@ -294,6 +294,18 @@ julia> x
  my_var[2]
 ```
 
+!!! tip
+    For models with millions of variables, setting the string name of each
```

Comment:
Why does the number of variables affect the proportion of the time spent in generating the names? Shouldn't the proportion remain constant as the model size increases, all else equal?

Reply:
I guess so. But it's really only noticeable as you get to larger models.